Dockerfile

```diff
@@ -38,7 +38,6 @@ COPY ./components/tipb_helper/Cargo.toml ./components/tipb_helper/Cargo.toml
 RUN mkdir -p ./src/bin && \
     echo 'fn main() {}' > ./src/bin/tikv-ctl.rs && \
     echo 'fn main() {}' > ./src/bin/tikv-server.rs && \
-    echo 'fn main() {}' > ./src/bin/tikv-importer.rs && \
```

Comment:
Have you tested the changes locally?

Reply:
This is the result of `gen-dockerfile.sh`...

Just tried `docker build .` and nothing special happened. What else to test?

Reply:
if you can build docker successfully, it is fine.